cocotb/decorators.py

```diff
@@ -87,6 +87,8 @@ class RunningTask:
         triggers to fire.
     """
 
+    _id_count = 0  # used by the scheduler for debug
```

Comment:
What are the merits of this over `id`?

Reply:
I think the main one is this from the docs:
> Two objects with non-overlapping lifetimes may have the same `id()` value.
But aesthetically I prefer a smaller integer and I think it's easier to match up visually when looking at log outputs.